src/javascript/lib/landing_pages/ico.js

```diff
@@ -37,7 +37,6 @@ window.onload = function() {
 
         if (!hash) {
             switchView('home');
-            scrollTo(0);
```

Comment:
why did we remove this?

Reply:
to avoid scrolling to the top of the page everytime a user refreshes the page

Reply:
was it requested? because we're doing this on our site, should we remove it there too?

Reply:
no, it wasn't requested and i don't think we're doing this on binary-static since the js and css for landing pages are sort of separated from binary-static

Reply:
so isn't it better to keep it and maintain consistency?